File: Lua/4DaysORM/src/Models/Position.lua
local string_values = {
    "Boss",
    "Manager",
    "Staffer"
}

local Position = {
    Boss = 1,
    Manager = 2,
    Staffer = 3,

    to_string = function (position)
        return string_values[position]
    end
}

return Position

File: Dart/orm/lib/position.dart
enum Position {
  staffer,
  manager,
  boss;

  @override
  String toString() {
    switch (this) {
      case Position.staffer:
        return "Staffer";
      case Position.manager:
        return "Manager";
      case Position.boss:
        return "Boss";
    }
  }
}
